Resources/config/exception_listener.xml

```diff
@@ -5,13 +5,22 @@
            xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
-        <service id="fos_rest.exception_listener" class="FOS\RestBundle\EventListener\ExceptionListener">
-            <tag name="kernel.event_subscriber" />
+        <service id="security.exception_listener" class="FOS\RestBundle\EventListener\ExceptionListener">
```

Comment:
No worries, slight issue though `security.exception_listener` is abstract so I can't appear to decorate it like this or pass it into a service as an argument.